ncm-metaconfig/src/main/pan/components/metaconfig/schema.pan

```diff
@@ -118,9 +130,32 @@ type ${project.artifactId}_config =  {
     'contents' : ${project.artifactId}_extension
     @{Predefined conversions from EDG::WP4::CCM::TextRender}
     'convert' ? ${project.artifactId}_textrender_convert
+    @{Actions (i.e. names found in /software/components/metadata/commands) to run when processing the service.
+      Refer to the ${project.artifactId}_actions type definition for the available hooks
+      for when a command may be run.}
```

Comment:
panlint is objecting to the indentation here: Indentation should be a multiple of four spaces

Reply:
this is a bug in panlint. i think it's confsued with closing `}` line before.

Reply:
The same bug seems to exist in the syntax highlighting.